individualmentoring

```diff
@@ -0,0 +1,101 @@
+# How to individually mentor a struggling student
+
+Context:
+
+Many of our students struggle during the JavaScript phase. If you want to mentor them on an individual basis, the folowing approach
+can help you mentor the student in a structured and efficient way. We've cut it up into 3 steps.
+
+##1.Asses
+
+The first step in making a plan for the students you are mentoring would be to assess the level of the student. 
+
+In assessing the skill level, we look for possession of the skillset taught in our curriculum, with as the most important skillset : fundamental knowledge of JavaScript. (and therewith fundamental knowledge of programming)
+
+The following general topics must be understood well with regard to JavaScript:
+
+Basics:
+
+⋅⋅*	Basic Data types [Strings, Numbers, Arrays, Booleans]
+⋅⋅* Operators
+⋅⋅*	Naming conventions
+⋅⋅* Conditions
+⋅⋅* Advanced data types [Objects]
+⋅⋅*	Statements vs Expressions
+⋅⋅* Loops (for/while)
+⋅⋅* Functions
+
+More advanced:
+
+⋅⋅*	Working with the debugger
+⋅⋅* Array Manipulations
+⋅⋅* Map and filter
+⋅⋅* Closures
+⋅⋅* Sound understanding of Scope
+⋅⋅* Callbacks
+⋅⋅* Events
+
+
+Advanced:
+
+⋅⋅* Understanding  of Codeflow
+⋅⋅* Async vs Sync
+⋅⋅* XHTTP requests
+⋅⋅* API calls
+⋅⋅* Rewrite data structures in Json
+⋅⋅* Promises
+
+
+The best way to asses these topics, is 
+
```

Comment:
I would start by seeing if the student is comfortable working with his environment . I often experienced students find running a file with node very difficult in the beginning of javascript. Also setting up a proper folder structure where they have all their homework, things they are working on can help. Most students have files all over their computer and multiple clones of the same repo, which does not make things easier.

Reply:
Bit of a hypothetical question, but would this be easy for a relatively new mentor to assess? The more experienced mentors would probably know what works for many beginners, but someone else might make suggestions that aren't really in line with what we have found out to be most helpful. Just wondering if this would require a best-practices guide. @mkruijt